crates/sargon/src/wrapped_radix_engine_toolkit/low_level/v2/transaction_manifest_v2/child_intents.rs

```diff
@@ -0,0 +1,93 @@
+use crate::prelude::*;
+
+#[derive(Clone, Debug, PartialEq, Eq, uniffi::Record)]
+pub struct ChildIntents {
+    pub children: Vec<ChildSubintent>,
+}
+
+impl From<ChildIntents> for Vec<ScryptoChildSubintent> {
+    fn from(value: ChildIntents) -> Self {
+        value.children.into_iter().map(Into::into).collect()
+    }
+}
+
+impl From<ChildIntents> for ScryptoChildIntents {
+    fn from(value: ChildIntents) -> Self {
+        ScryptoChildIntents {
+            children: value.into(),
+        }
+    }
+}
+
+impl From<(Vec<ScryptoChildSubintent>, NetworkID)> for ChildIntents {
+    fn from(value: (Vec<ScryptoChildSubintent>, NetworkID)) -> Self {
+        Self {
+            children: value
+                .0
+                .into_iter()
+                .map(|c| (c, value.1).into())
+                .collect(),
+        }
+    }
+}
+
+impl ChildIntents {
+    pub(crate) fn empty() -> Self {
```

Comment:
can also derive `Default` , and then use `ChildIntents::default()` instead of this. But I like named `empty`.. maybe `empty` can use `Self::default()`...